mithril-common/src/digesters/cardano_immutable_digester.rs

```diff
@@ -65,16 +65,16 @@ impl ImmutableDigester for CardanoImmutableDigester {
                 })
             }
             Some(_) => {
-                info!(self.logger, "#compute_digest"; "beacon" => #?beacon, "nb_of_immutables" => immutables.len());
+                info!(self.logger, ">> compute_digest"; "beacon" => #?beacon, "nb_of_immutables" => immutables.len());
```

Comment:
I think this log should be located at the beginning of the function, for example on line 53 of the file.